baybe/searchspace/discrete.py

```diff
@@ -206,6 +208,12 @@ def from_dataframe(
         Raises:
             ValueError: If several parameters with identical names are provided.
             ValueError: If a parameter was specified for which no match was found.
+
+        Note:
+            For columns that have no corresponding entry provided via `parameters`,
```

Comment:
is this not redundant with the docstring already present for `parameters` above?

in a way this is telling me `if not specified, defaults will be used`... isnt that obvious? 
one could also live without the note by simply adding `(with default arguments)` to the docstring above

Reply:
Hm, not sure actually. The reason why I added it is because I just thought about it for the first time when I tested roundtrip serialization and noticed that the optional arguments were (of course) not set. And this is even though I was the one who wrote the method 😄 @AVHopp: what do you think? Is the note helpful or unnecessary since clear from the context?

Reply:
I understand the note basically as "If you do not provide additional arguments, defaults are used", so this should be clear in general. I however also see the point that it might make sense to make people aware of that.
I'd suppose to get rid of the second sentence (that feels really redundant to me) and include the note about the default values in the `Args` section for `parameters`. If that makes the `Args` section too packed, I'd also rather remove the note.

Reply:
Resolved, for details, see [here](https://github.com/emdgroup/baybe/pull/86#discussion_r1464858527).